src/python/WMCore/WMRuntime/Scripts/SetupCMSSWPset.py

```diff
@@ -5,190 +5,119 @@
 
 """
 from __future__ import print_function
-from future.utils import viewitems
 
```

Comment:
```suggestion
from builtins import next, object
```

These will come handy later